### ISSUE TITLE
Remove 'autoHide' prop from <Toast /> component

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#54]
automatically hiding content without giving users means of extending/turning off the auto hiding violates WCAG so we should not support it

**Comment by schne324** [comment]
resolves by #95
